README.md

```diff
@@ -1,21 +1,73 @@
+
+Up to this point, we've been able to create simple Ruby programs by using the `puts` method to output a string to the command line. In this codealong, we'll use the `gets` method to get data from the user and assign it to a variable for future use.
 
 ## Objectives
 
-1. Objective 1
-2. Objective 2
+1. Use the `gets` method to get data from the user.
+2. Assign the return value of `gets` to a variable
+3. Understand that `gets` always returns a string.
+3. Show that gets has a trailing space, so we usually use `.chomp` to remove it.
+
+## Using `gets`
+
+We're going to create a simple program called `greeter.rb`, that takes in a name from a user, and `puts` a welome phrase based on their input. Start by creating a new ruby file called `greeter.rb` and prompt the user for their name:
+
+```ruby
+puts "What is your name?"
+```
+Run the program to ensure that your program works.
+
+In order to create a prompt in Ruby, use the `gets` method, like so:
+
+```ruby
+puts "What is your name?"
+gets
+```
+Now when you run your program, it will pause until you fill in the prompt and hit the return key. Your program should then end.
+
+We now have a prompt, but the value that the user has given us isn't being saved for us to use. Why don't we use a variable to hold the data that our user is giving us? That way we can use the data further on in the program:
+
+```ruby
+puts "What is your name?"
+name = gets
+puts "Welcome to the Flatiron School, " + name + "!"
+```
+Run your program again. You'll now be able to input your name, and will get back a welcome phrase that looks something like this:
+
+```
+Welcome to the Flatiron School, Danny
+!
+```
+### Trailing Newlines 
+Do you notice anything weird about the output of the string? The "!" is on the next line! The reason we see this is that when we use the gets method to input our name, we have to hit `return` to move on to the next line of the program. This `return` is actually counted as part of the string we're saving to the `name` variable. This is called a "trailing newline", and has an easy fix. We use the `.chomp` method to remove trailing newline characters:
+
+```ruby
+puts "What is your name?"
+name = gets.chomp
+puts "Welcome to the Flatiron School, " + name + "!"
+```
+Run your program again - it should work!
+
 
-## Instructions
+### Gets with Numbers
+**Note**: `gets` will always return a string, regardless of whether the user is inputting numbers or letters in the prompt. As such, if you're planning on using the input as an integer or a float, you'll need to convert the input to the correct data type, using `.to_i` for integers and `.to_f` for floats. Check out this "Addition program" that uses `.to_i` to convert the user input to integers for addition:
 
+```ruby
+puts "Welcome to the addition machine. Please input your first number:"
+number1 = gets.to_i
+puts "Input your second number:"
+number2 = gets.to_i
+sum = number1 + number2
+puts "The sum of the two numbers is " + sum.to_s
```

Comment:
Maybe break converting the sum back into a string into a separate section? ie `puts "THe sum of the two numbers is " + sum` gives us a `TypeError: String can't be coerced into Fixnum`. This is because we're trying to add the integer `sum` with a string. To be able to concatenate the number to the end of the string, we need to first convert sum into a string...